### DISCUSSION TITLE
About pool_ contract_ The impact of wrong address

### DISCUSSION THREAD

**Discussion by youyusi** [Chia-Network/chia-blockchain#8621] [Q&A]
I have multiple wallets. When drawing, I accidentally made a wrong pool_ contract_ Address, it's really careless. There are 4P farmland.

I tried to modify the config.yml file, pool_ list>pooling>p2_ singleton_ puzzle_ hash。 Although the number of farmland displayed on the GUI is correct, the log keeps warning me, error in pooling: (9, 'invalid pool contract puzzle hash.

Although there is no way to use pooling normally, I want to know if I can use solo normally and get rewards without making any changes?

**Comment by loppefaaret** [comment]
if you used a wrong key to plot a plot, is it basically worthless - and all the data is based on those keys you provide, so there is nothing to do but replacing the plot

**Comment by BrandtH22** [comment]
@youyusi , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
